.github/workflows/acceptance.yml

```
@@ -0,0 +1,182 @@
+          - 3.0.2
+        include:
+          - { meterpreter: { name: windows_meterpreter }, os: windows-2022 }
+        exclude:
+          - { meterpreter: { name: mettle }, os: windows-2019 }
+          # Meterpreter is broken on Windows with PHP 5.3 https://github.com/rapid7/metasploit-payloads/issues/532
+          - { meterpreter: { name: php, runtime_version: 5.3 }, os: windows-2019 }
+          # Skip mac/ubuntu windows_meterpreter test runs
+          - { meterpreter: { name: windows_meterpreter }, os: macos-10.15 }
+          - { meterpreter: { name: windows_meterpreter }, os: ubuntu-18.04 }
+
+    runs-on: ${{ matrix.os }}
+
+    timeout-minutes: 25
+
+    env:
+      RAILS_ENV: test
+      METERPRETER: ${{ matrix.meterpreter.name }}
+      METERPRETER_RUNTIME_VERSION: ${{ matrix.meterpreter.runtime_version }}
+
+    name: ${{ matrix.meterpreter.name }} ${{ matrix.meterpreter.runtime_version }} ${{ matrix.os }}
+    steps:
+      - name: Install system dependencies (Linux)
+        if: runner.os == 'Linux'
+        run: sudo apt-get install libpcap-dev graphviz
+
+      - uses: shivammathur/setup-php@15c43e89cdef867065b0213be354c2841860869e
+        if: ${{ matrix.meterpreter.name == 'php' }}
+        with:
+          php-version: ${{ matrix.meterpreter.runtime_version }}
+          tools: none
+
+      - name: Set up Python
+        if: ${{ matrix.meterpreter.name == 'python' }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.meterpreter.runtime_version }}
+
+      - uses: actions/setup-java@v2
+        if: ${{ matrix.meterpreter.name == 'java' }}
+        with:
+          distribution: temurin
+          java-version: ${{ matrix.meterpreter.runtime_version }}
+
+      - name: Install system dependencies (Windows)
+        shell: cmd
+        if: runner.os == 'Windows'
+        run: |
+          REM pcap dependencies
+          powershell -Command "[System.Net.ServicePointManager]::ServerCertificateValidationCallback = {$true} ; [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; (New-Object System.Net.WebClient).DownloadFile('https://www.winpcap.org/install/bin/WpdPack_4_1_2.zip', 'C:\Windows\Temp\WpdPack_4_1_2.zip')"
+
+          choco install 7zip.installServerCertificateValidationCallback
+          7z x "C:\Windows\Temp\WpdPack_4_1_2.zip" -o"C:\"
+          REM mv C:\WpdPack\Lib\x64\Packet.lib C:\WpdPack\Lib\
+          REM mv C:\WpdPack\Lib\x64\wpcap.lib C:\WpdPack\Lib\
+
+          dir C:\\
+
+          dir %WINDIR%
+          type %WINDIR%\\system32\\drivers\\etc\\hosts
+
+          REM php configuration - enable sockets, otherwise php reverse shells fail on AF_INET constants
+          php -r "file_put_contents(php_ini_loaded_file(), PHP_EOL . 'extension=sockets' . PHP_EOL, FILE_APPEND);"
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Ruby
+        env:
+          BUNDLE_WITHOUT: "coverage development pcap"
```

Potentially:

```suggestion
          BUNDLE_WITHOUT: "coverage development"
```

Or:

```
          BUNDLE_FORCE_RUBY_PLATFORM: 1
```

Or:

```
group :development, :test do
  # running documentation generation tasks and rspec tasks
  gem 'rake'
  # Define `rake spec`.  Must be in development AND test so that its available by default as a rake task when the
  # environment is development
  gem 'rspec-rails'
  gem 'rspec-rerun'
  gem 'rubocop'
end

group :test do
  # automatically include factories from spec/factories
  gem 'test-prof'
  gem 'factory_bot_rails'

  gem 'allure-rspec'
  # Make rspec output shorter and more useful
  gem 'fivemat'
end
```